Sources/LiveViewNative/Modifiers/Search Modifiers/FindNavigatorModifier.swift

```diff
@@ -22,15 +33,15 @@ import SwiftUI
 #endif
 @available(iOS 16.0, *)
 struct FindNavigatorModifier: ViewModifier, Decodable {
-    /// A binding that controls whether the find navigator is shown.
+    /// A value that controls whether the find navigator is shown.
     #if swift(>=5.8)
     @_documentation(visibility: public)
     #endif
-    @LiveBinding private var isPresented: Bool
+    @ChangeTracked private var isPresented: Bool
     
     init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
This produces the following warning now that `container` is unused: `Initialization of immutable value 'container' was never used; consider replacing with assignment to '_' or removing it`